### DISCUSSION TITLE
SimpleHttpOperator is not catching response for long running tasks > 20 min

### DISCUSSION THREAD

**Discussion by Arti3DPlayer** [apache/airflow#24781] [Q&A]
### Apache Airflow version

2.3.2 (latest released)

### What happened

When I use Dag with simple http operator and task takes more than 20 min, it is not catching returned response and working forever

```
SimpleHttpOperator(
            task_id=f'simple_http',
            http_conn_id='api',
            method='POST',
            data=json.dumps({}),
            endpoint=f'/api/test/',
            headers=headers,
            log_response=True,
            response_check=lambda response: response.status_code == 200,
        )
```

I tried to increase `killed_task_cleanup_time` and `decrease worker_concurrency`, but this not helped. Also I found 2 same issues:

https://stackoverflow.com/questions/69582817/no-response-from-request-apilonger-than-30min-while-using-simplehttpoperator

https://github.com/apache/airflow/discussions/21365

### What you think should happen instead

Operator must ended with success or failure based on response.

### How to reproduce

Call an API endpoint that with sleep 30 min.

### Operating System

linux

### Deployment

Composer

### Deployment details

Google composer

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by eladkal** [comment]
As already pointed out in previous discussion https://github.com/apache/airflow/discussions/21365#discussioncomment-2120746 this is unlikely to be Airflow issue. This report doesn't provide any insight into why do you think this is an Airflow bug?

Since this report is identical to https://github.com/apache/airflow/discussions/21365 I'm locking the conversation here. Lets continue on https://github.com/apache/airflow/discussions/21365
